docs/source/users/index.md

```diff
@@ -0,0 +1,137 @@
+# Users
+
+This page is intended for people interested in installing and using Jupyter AI. If you are interested in contributing a new or modified feature in JupyterLab, please see our {doc}`contributor's guide </contributors/index>`.
+
+## Prerequisites
+
+To use Jupyter AI, you will need to have JupyterLab ≥ 3.5 (*not* JupyterLab 4) installed.
```

Comment:
`need to have JupyterLab ≥ 3.5 (*not* JupyterLab 4) installed` is somewhat confusing to me. Would something like `need to have JupyterLab version 3.5 or above (but below version 4.0.0)`? Or use yarn standard notation `need to have JupyterLab ≥3.5.0 <4.0.0`?